scripts/install-windows-prereqs.ps1

```diff
@@ -561,7 +568,19 @@ try {
     Install-OCaml
     Install-Shellcheck
     Install-PSW
-    if ($WithFLC -eq $true)
+    
+    Write-Host "WithAzureDCAPClient option: $WithAzureDCAPClient"
+    if ($WithAzureDCAPClient -eq "azure")
+    {
+        Write-Host "*** Installing Azure.DCAP.Windows ***"
+        Install-AzureDCAPWindows 
+    }
+    else
+    {
+        Write-Host "*** NOT installing Azure.DCAP.Windows ***"
```

Comment:
Let's change this to: "Installing no DCAP client."

Reply:
I changed it to NOT installing Azure.DCAP.Windows client.

Reply:
I'd prefer if this didn't reference Azure in the situation where someone doesn't specify it! It's important to keep OE cloud-neutral, in my opinion